src/components/ProjectSearch/TextSearch.js

```diff
@@ -74,43 +71,46 @@ export default class TextSearch extends Component {
     if (focused) {
       this.focused = { setExpanded, file, expanded };
     }
-    return dom.div(
-      {
-        className: classnames("file-result", { focused }),
-        key: file.id,
-        onClick: e => setExpanded(file, !expanded)
-      },
-      Svg("arrow", {
-        className: classnames({
-          expanded
-        })
-      }),
-      Svg("file"),
-      dom.span({ className: "file-path" }, getRelativePath(file.filepath)),
-      dom.span(
-        { className: "matches-summary" },
-        ` (${file.matches.length} match${file.matches.length > 1 ? "es" : ""})`
-      )
+    const Arrow = Svg("arrow");
+    const File = Svg("file");
+    return (
+      <div
+        className={classnames("file-result", { focused })}
+        key={file.id}
+        onClick={e => setExpanded(file, !expanded)}
+      >
+        <Arrow
+          className={classnames({
+            expanded
+          })}
+        />
+        <File />
+        <span className="file-path">
+          {getRelativePath(file.filepath)}
+        </span>
+        <span className="matches-summary">
+          {` (${file.matches.length} match${file.matches.length > 1
```

Comment:
lets move this up to a variable. I'm still not comfortable with JSXy expressions